Program.cs

```diff
@@ -1,4 +1,26 @@
-﻿using System;
-Console.WriteLine ("Hello, World!");
-
-   
\ No newline at end of file
+﻿// --------------------------------------------------------------------------------------------
+// Training ~ A training program for interns at Metamation, Batch - July 2023
+// Copyright (c) Metamation India.                                              
+// ------------------------------------------------------------------------
+// Program.cs
+// Extension of Spelling Bee assignment
+// Display the first 7 letters to be used as the seed for the Spelling Bee program from the given word list.
+// --------------------------------------------------------------------------------------------
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+// Reading all letter in the wordlist.
+string wordsList = File.ReadAllText ("D:\\OneDrive - Trumpf Metamation Pvt Ltd\\words.txt");

```

Comment:
Until we start using embedded resources, use the location `C:/etc/words.txt`. Notice the use of forward slash. This is more readable than using double backslash.